### ISSUE TITLE
Conversation

### ISSUE THREAD

**Issue by cinnamoner** [cinnamoner/Blank#1]

**Comment by cinnamoner** [comment]
@feren Are you there?

**Comment by feren** [comment]
@cinnamoner Yup.

**Comment by cinnamoner** [comment]
@feren Okay. Are you particularly busy at the moment? Cause I need help.

**Comment by feren** [comment]
@cinnamoner ?

**Comment by cinnamoner** [comment]
@feren I won't be upset if you don't wanna help or if you're not into that mood. With that being said there it goes:

Prerequisites:
Windows
Any browser 
Avast running

Try downloading Neon User or User LTS. Avast should block the download. Take a shot with Avast poping up a message sayin *threat detected*. Then go to the KDE forum. Open an account and post the shot. In the KDE Neon section. Topic: *Avast blocks Neon images*. That would be it. It would mean a lot to me. Why is it so important? I'll explain later. Can/do you want to do it? You will find me on that forum.

**Comment by feren** [comment]
@cinnamoner Yeah, hang on then, watching a video by EverythingApplePro on YT, though, it may look UGLY AF because of High Contrast (lenny)

**Comment by cinnamoner** [comment]
https://forum.kde.org/

**Comment by feren** [comment]
@cinnamoner Just thinking about it now, won't they just say report it as a False Positive to Avast?

PS: To do so: Virus Warning > More Details > Report as False Positive

**Comment by cinnamoner** [comment]
@feren Yes, it was reported as a false positive by me and seemingly the admin at KDE forum. Avast hasn't solved the issue. I wanned one more proof there's nothing wrong on my computer and Avast is crap. So I can dump Avast and replace it. Avast acknowledged those were false positives but nothing happened although they said they fixed the problem.

**Comment by cinnamoner** [comment]
@feren I've got 3 posts on that forum. Please read them with some attention.

**Comment by cinnamoner** [comment]
I need proof there's nothing wrong with my computer and I need proof a third party (you) can replicate the event. Avant will block the download when all most complete.

**Comment by cinnamoner** [comment]
@feren So?

**Comment by cinnamoner** [comment]
@Elbullazul Welcome Christian!

**Comment by feren** [comment]
@cinnamoner No, haven't found 'em

**Comment by feren** [comment]
Wait, can't I just download them outside of Windows, and then make them accessible in Windows for Avast to find 'em?

**Comment by cinnamoner** [comment]
I don't know why Neon doesn't have Damn SHA on a secure server. The pgp is usless for Windows user. And I don't care about Cleopatra. Do you understand where I'm commin from?

**Comment by cinnamoner** [comment]
@feren You haven't found them because you haven't opened an account. It's invisible to you.

**Comment by cinnamoner** [comment]
That's okay too @feren Download the iso in Linux. Extract vmlinuz and vmlinuz.efi. Then submit both files to Virus Total and post the result here.

**Comment by cinnamoner** [comment]
@feren Do it in any way you want. Jusy want to know wheter Avast finds Win64:vitro in those files again.

**Comment by cinnamoner** [comment]
@feren Sit back. Relax. Think. Act. Tell.

**Comment by feren** [comment]
BTW, warned Quidsup about it, left a comment on his latest WOLN video about it

**Comment by cinnamoner** [comment]
@feren About what exactly have you warned Quid? That wasn't what I asked for.

Fact: Avast blocks Neon images. The false positives have been submitted to Avast. They said those are false positives but failed to remedy the situation. That's all I know. Do I need to write a book about it?

**Comment by cinnamoner** [comment]
@feren I asked you to find out wheter you can replicate a situation. Can you do that?
